parm/config/config.fcst

```diff
@@ -5,10 +5,26 @@
 
 echo "BEGIN: config.fcst"
 
+# set -eu
+
 # Source model specific information that is resolution dependent
 . $EXPDIR/config.fv3 $CASE
 [[ "$CDUMP" == "gfs" ]] && export nth_fv3=$nth_fv3_gfs
 
+# Turn off waves if not used for this CDUMP
+case $WAVE_CDUMP in
+  both | $CDUMP ) ;; # Don't change
+  *) DO_WAVE="NO" ;; # Turn waves off
+esac
+
+# Source component configs if necessary
+for component in WAVE OCN ICE AERO; do
```

Comment:
Why is this not done in the job scripts and instead hidden here?

Reply:
j-jobs are supposed to be as clean as possible, and there is the extra bit of stuff to determine if waves are on before this. Kate & I both agree this is just a better place for it.

Reply:
I agree w/ @JessicaMeixner-NOAA 
This should be part of the j-job where all the configs are sourced.
Adding this block is not really polluting the j-job.

Reply:
It's not just this block. There is a preceding block that changes DO_WAVE based on CDUMP and WAVE_CDUMP, so four more lines.

Also, config.fv3 is already sourced here first and pre-exists this change. We can move the whole lot to the j-job if that's what we really want to do, but moving here is consistent with what was here before. (config.fv3 should also be renamed config.atm to be consistent with the others.)

Given these, I would recommend keeping it as-is for now and then deciding whether we want to move the whole thing to the j-job for a future PR.

Reply:
So this is going to stay as is?  I still think this is now being hidden and becomes confusing for users.

Reply:
After discussing with Rahul, we decided it should stay as-is.

Reply:
I don't see any of that discussion here

Reply:
We discussed it on the g-w tag-up a couple weeks ago, but it was just repeating/elaborating on my points here.